### ISSUE TITLE
math.options.precision doesn't act consistently or according to def of precision

### ISSUE THREAD

**Issue by jmichae3** [josdejong/mathjs#81]
``` javascript
//must initialize units with a same-kind unit, lbm is lbs mass
totalweight=math.eval("0 lbm"); 
weight=math.eval(document.getElementById(id).value).in('lbm');
totalweight = totalweight.add(weight);
//final output
math.options.precision=4;//formats output to 1 digit after decimal point.
document.getElementById('out').innerHTML=totalweight.toString();
```

I am getting results I can't explain with the above math.options.precision=4;
it's giving me output like:
0.3159 lbm
0.463 lbm
0 lbm
0.8 lbm
45.4 lbm
1.63 lbm
3.042 lbm

is this only partially implemented (too early yet?)? the only place I can figure where precision is handled is in the functions exports.toPrecision(value, precision)

my real code (it's the weight column):
http://jesusnjim.com/calculators/PSU-calculators/power-supply-load-calculator.html#app
around lines 2704-2781 (this may change as I add/remove products)

see definition of precision here:
http://www.merriam-webster.com/dictionary/precision
basically, it's the number of digits past the decimal point.

am I maybe too early for the party again because format is not implemented and string depends upon format?

I have a workaround which I may need to use in a few minutes so things don't get too wide. .toFixed() is a js method of the Number object.

``` javascript
document.getElementById(id).innwerHTML=totalweight.toNumber('lbm').toFixed(2);
```

this gives you 2 decimal places after the decimal point.
